clang/lib/Basic/Targets/NVPTX.h

```diff
@@ -75,6 +75,8 @@ class LLVM_LIBRARY_VISIBILITY NVPTXTargetInfo : public TargetInfo {
 
   ArrayRef<Builtin::Info> getTargetBuiltins() const override;
 
+  bool useFP16ConversionIntrinsics() const override { return false; }
```

Comment:
This hook should have been deleted years ago and the intrinsics should be removed

Reply:
Agreed, but figured this is easier to get this working in the short-term.